userapp/views.py

```diff
@@ -0,0 +1,81 @@
+from django.shortcuts import render
+from rest_framework import viewsets
+from userapp.models import User, Account, Transactions
+from userapp.serializer import UserSerializer, AccountSerializer, TransactionSerializer
+from rest_framework.response import Response
+from userapp.services import deposit, create_acc_no, withdraw
+# Create your views here.
+
+
+def index(request):
+    return render(request, 'base.html')
+
+
+class UsersView(viewsets.ViewSet):
+    serializer_class = UserSerializer
+    queryset = User.objects.all()
+
+    def list(self, request):
+        queryset = User.objects.all()
+        serializer = UserSerializer(queryset, many=True)
+        return Response(serializer.data)
+
+    def update(self, request, pk):
+        queryset = User.objects.get(pk=pk)
+        serializer = UserSerializer(queryset)
+        return Response(serializer.data)
+
+    def create_user(self, request):
```

Comment:
There should be consistency in naming. The update function is called just 'update' while the create function is called 'create_user'. Update is the right way to go as it overrides parents method with the same name.